README.md

```diff
@@ -29,9 +29,9 @@ const SecretStack = require('secret-stack')
 const caps = require('ssb-caps')
 const {and, type, author, toCallback} = require('ssb-db2/operators')
 
-const sbot = SecretStack({appKey: caps.shs})
+const sbot = SecretStack({ caps })
```

Comment:
Does this work like this? (I know mix commented already)
I often use `SecretStack({appKey: require('ssb-caps').shs})` but I would **love** to use `SecretStack({ caps: require('ssb-caps') })`. Basically just asking if you're sure it works

Reply:
That's how ssb-server does it https://github.com/ssbc/ssb-server/blob/53c29aa819ad82b068c0b8e25db664215d1b2734/index.js#L6

Reply:
TIL